content/articles/dns-query-limits.md

```diff
@@ -17,53 +19,32 @@ TL;DR: We charge $0.10 per million queries per zone on Solo and Teams plans.
 
 ---
 
-## How we calculate fees
-
-- The price for up to, but not including, one million queries per zone is $0.10.
-- The DNS query volume fees are calculated at this price every time the registered volume goes beyond any given million-query mark (starting at 0 queries):
-    - The fee for 1 - 999,999 queries is $0.10
-    - The fee for 1M - 1,999,999 queries is $0.20
-    - The fee for 2M - 2,999,999 queries is $0.30
-    - And so on.
-- We won't charge fractions of the price:
-    - Any volume greater or equal to 1M and less than 2M will have a fee of $0.20
-    - For example, the fee for 1.5M queries is still $0.2.
-
-Let's explore this using a couple examples.
-
-### Example 1: 2,000,000 total queries across 4 zones
```

Comment:
This was written before we had a pricing calculator, and the examples get a little convoluted. I removed them entirely and just added a link to the calculator so people can get an actual estimate of their own volumes instead of trying to parse our hypotheticals. But, if there's a reason to add an example back, we can do that.

Reply:
I think it's less confusing to just have the tooltip show the beginning portion with the pricing breakdown and let the customer enter their own. 

![Screenshot 2024-06-03 at 1 00 38 PM](https://github.com/dnsimple/dnsimple-support/assets/42185139/27ceb061-db86-4241-a375-7b8d49bdcbef)

If we want to keep these examples, they'll need to be edited. The way this was copied over, it's missing at least one period, and the headers are different which should have resulted in the copy being tweaked (I didn't know this existed until now).

Reply:
Ok thanks, im happy to work with you to make this screen more clear.